evap/results/exporters.py

```diff
@@ -101,11 +106,18 @@ def filter_text_and_heading_questions(questions):
         return filtered_questions
 
     @staticmethod
-    def filter_evaluations(semesters, evaluation_states, degrees, course_types, contributor, include_not_enough_voters):
+    def filter_evaluations(
+        semesters: Iterable[Semester],
+        evaluation_states: Iterable[int],
```

Comment:
`Iterable[Evaluation.State]`?

Reply:
Yeah, nice try, but `Evaluation.State.X` is an integer and so we cannot type this as a list because it looses all context information.

Reply:
In that case, we should make `Evaluation.State` an `IntEnum` in the future ([the fsm field supports that](https://github.com/viewflow/django-fsm/issues/204), doesn't require a migration, but juts quickly patching it breaks some UI, e.g. the staff semester index view). For now, I'd still like to have something else than `int` here because the type very clearly is not `int`. Passing `1337` is never correct. We can use `Any` and add a follow-up issue to make it an `IntEnum` and use the correct type here. We can also use some placeholder type for now, which can be equal to `int` or `Any`, but add semantical clearness with a different name (`StateT` or whatever)

@niklasmohrin thoughts?

Reply:
Decided to use `Any`

Reply:
After #2174, this should now be possible :)
```suggestion
        evaluation_states: Iterable[Evaluation.State],
```